omod-2.0/src/main/java/org/openmrs/module/attachments/rest/AttachmentResource2_0.java

```diff
@@ -1,17 +1,205 @@
+			
+		} else if (rep instanceof FullRepresentation) {
+			model.property("preferredName", new RefProperty("#/definitions/AttachmentNameGet"))
+			        .property("preferredAddress", new RefProperty("#/definitions/AttachmentAddressGet"))
+			        .property("names", new ArrayProperty(new RefProperty("#/definitions/AttachmentNameGet")))
+			        .property("addresses", new ArrayProperty(new RefProperty("#/definitions/AttachmentAddressGet")));
+		}
+		return model;
+	}
+  
+	@Override
+	public Model getCREATEModel(Representation representation) {
+		ModelImpl model = new ModelImpl()
+		        .property("names", new ArrayProperty(new RefProperty("#/definitions/AttachmentNameCreate")))
+		        .property("gender", new StringProperty()._enum("M")._enum("F")).property("age", new IntegerProperty())
+		        .property("birthdate", new DateProperty())
+		        .property("birthdateEstimated", new BooleanProperty()._default(false))
+		        .property("dead", new BooleanProperty()._default(false)).property("deathDate", new DateProperty())
+		        .property("causeOfDeath", new StringProperty())
+		        .property("addresses", new ArrayProperty(new RefProperty("#/definitions/AttachmentAddressCreate")))
+		        .property("attributes", new ArrayProperty(new RefProperty("#/definitions/AttachmentAttributeCreate")));
+		
+		model.setRequired(Arrays.asList("names", "gender"));
+		return model;
+	}
+	
+	@Override
+	public Model getUPDATEModel(Representation representation) {
+		return new ModelImpl().property("dead", new BooleanProperty()).property("causeOfDeath", new StringProperty())
+		        .property("deathDate", new DateProperty()).property("age", new IntegerProperty())
+		        .property("gender", new StringProperty()._enum("M")._enum("F")).property("birthdate", new DateProperty())
+		        .property("birthdateEstimated", new BooleanProperty()._default(false))
+		        .property("preferredName", new StringProperty().example("uuid"))
+		        .property("preferredAddress", new StringProperty().example("uuid"))
+		        .property("attributes", new ArrayProperty(new RefProperty("#/definitions/AttachmentAttributeCreate")))
+		        
+		        .required("dead").required("causeOfDeath");
+	}
+	
+	/**
+	 * @see org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResource#getPropertiesToExposeAsSubResources()
+	 */
+	@Override
+	public List<String> getPropertiesToExposeAsSubResources() {
+		return Arrays.asList("comment", "complexData", "Attributes");
+	}
+	
+	@Override
+	public Attachment newDelegate() {
+		return new Attachment();
+	}
+	
+	@SuppressWarnings("deprecation")
+	@Override
+	public Attachment getByUniqueId(String uniqueId) {
+		Obs obs = Context.getObsService().getObsByUuid(uniqueId);
+		if (!obs.isComplex())
+			throw new GenericRestException(uniqueId + " does not identify a complex obs.", null);
+		else {
```

Comment:
```suggestion
		if (!obs.isComplex()) {
			throw new GenericRestException(uniqueId + " does not identify a complex obs.", null);
		} else {
```
You forgot the curly braces here.
Ref.: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-CodeStyle